Data/Models/AffixData.cs

```diff
@@ -0,0 +1,90 @@
+﻿using PathOfTerraria.Core;
+using System.Collections.Generic;
+
+namespace PathOfTerraria.Data.Models;
+
+public class ItemAffixData
+{
+	public class TierData
+	{
+		public float MinValue { get; set; }
+		public float MaxValue { get; set; }
+		public int MinimumLevel { get; set; }
+		public float Weight { get; set; }
+
+		public override string ToString()
+		{
+			return $"Value Range: {MinValue}-{MaxValue}, Min. Level: {MinimumLevel}, Weight: {Weight}";
+		}
+	}
+
+	public string AffixType { get; set; }
+	public string EquipTypes { get; set; }
+	public string Influences { get; set; }
+	public List<TierData> Tiers { get; set; }
+
+	public TierData GetAppropriateTierData(int level)
```

Comment:
Wasn't this supposed to be a random tier allowed by on itemLevel and based on weight?

Reply:
I don't think I actually ended up using that, I have a method on the AffixRegistry that has this

Reply:
In RollAffixes you use GetRandomAffixValue that uses that.